node/cn/api_backend.go

```diff
@@ -278,6 +278,8 @@ func (b *CNAPIBackend) ProtocolVersion() int {
 	return b.cn.ProtocolVersion()
 }
 
+// If current block is magma hard forked, it returns the baseFee * 2.
```

Comment:
For the convention of goDoc comments https://go.dev/blog/godoc
```suggestion
// SuggestPrice returns the baseFee * 2 if the current block is magma hard forked.
```